lib/DriverTool/swift_symbolgraph_extract_main.cpp

```diff
@@ -203,6 +203,25 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args,
             .Default(AccessLevel::Public);
   }
 
+  if (auto *A = ParsedArgs.getLastArg(OPT_allow_availability_platforms)) {
+    llvm::SmallVector<StringRef> AvailabilityPlatforms;
+    StringRef(A->getValue())
+        .split(AvailabilityPlatforms, ',', /*MaxSplits*/ -1,
+               /*KeepEmpty*/ false);
+    Options.AvailabilityPlatforms = llvm::DenseSet<StringRef>(
+        AvailabilityPlatforms.begin(), AvailabilityPlatforms.end());
+    Options.AvailabilityIsBlockList = false;
+  } else if (auto *A =
+                 ParsedArgs.getLastArg(OPT_block_availability_platforms)) {
```

Comment:
The way this is written, it will take any "allow" even if there is a "block" coming later, which is unexpected.  I'd recommend using the getLastArg form that takes both option names, so you get the last of either, and then set the Boolean based on which flag you got.

Reply:
I was wondering if there was a better way to go about it. Thanks for the suggestion! I can apply this here and forward-port it back to main afterward if that's okay.